### PR TITLE
using code snippet in https://cloud.google.com/pubsub/docs/emulator

### PR DESCRIPTION
**Pull request by kenfesta**
Adding region tags for use in https://cloud.google.com/pubsub/docs/emulator

### CODE REVIEW

**Comment by jskeet** [comment]
The text from these snippets is produced *directly* in https://googleapis.github.io/google-cloud-dotnet/docs/faq.html. This pull request would start including those comments in our documentation, which we wouldn't want.

If we want to use the same samples across the two documentation systems, we'll need to modify at least one of the systems - we can't use the existing syntax for both systems in a single example.

**Comment by jskeet** [comment]
A note on this - it would be *relatively* easy to change our system to just ignore any line containing "// [START" or "// [END". That may be simpler than changing the GCP-wide system. Let me know if you want me to take that on.

File: tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs [hunk 1]
```diff
@@ -37,6 +38,7 @@ public void Emulator()
             {
                 Console.WriteLine(topic.Name);
             }
+            // [START pubsub_use_emulator]
```
Comment (jskeet) [inline tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs]: Presumably this is intended to be END rather than START

**Comment by jskeet** [comment]
(Travis failure is a failure to install .NET Core - nothing to do with this PR)

File: tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs (hunk 1 above)
Comment (kenfesta) [reply tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs]: Yes, thanks!

Comment (jskeet) [reply tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs]: I've merged a tool change, so once this is fixed I'll be able to merge.

**Comment by jskeet** [comment]
What's the status of this? I'd like to either merge it after the minor fix, or close it if it's not going to be useful to you for a while.

**Comment by jskeet** [comment]
What's the status of this? I'd like to either merge it after the minor fix, or close it if it's not going to be useful to you for a while.

**Comment by kir-titievsky** [comment]
Reviewers: any pointers on getting this merged? Something we can do?

**Comment by jskeet** [comment]
@kir-titievsky: Yes, I'm just waiting for the requested typo change to be made.

**Comment by kenfesta** [comment]
Will create a pull request directly from the GUI editor.

**Pull request closed without merging**